.codecov.yml

```diff
@@ -0,0 +1,10 @@
+coverage:
+  status:
+    project:
+      default:
+        target: 0%
```

Comment:
Now this is production aimed, can it have a target; even if we say 50% (or less) for now and stick in a card to increase coverage?